pkg/test/vpc_client/route.go

```diff
@@ -0,0 +1,44 @@
+package vpc_client
```

Comment:
to keep consistent, we should rename this to `route_table.go` as it is in `/aws/`